locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+
+#: src/screens/DashboardTx.js:127
+msgid "Blocks |fn:(See all blocks)|"
+msgstr "Blocos |fn:(Ver todos)|"
+
+#: src/screens/DashboardTx.js:134
+msgid "Transactions |fn:(See all transactions)|"
+msgstr "Transações |fn:(Ver todas)|"
+
+#: src/components/Navigation.js:47 src/screens/DashboardTx.js:154
+msgid "Explorer"
+msgstr "Explorer"
+
+#: src/screens/DashboardTx.js:155
+msgid ""
+"Here you can see the most recent transactions and blocks of the network."
+msgstr ""
+"Aqui você pode visualizar os blocos e transações mais recentes da rede."
+
+#: src/components/TokenHistory.js:242 src/screens/DashboardTx.js:161
+msgid "ID"
+msgstr "ID"
+
+#: src/screens/DashboardTx.js:162
+msgid "Timestamp"
+msgstr "Data e Hora"
+
+#: src/screens/DecodeTx.js:91
+msgid "Decode tx"
+msgstr "Decodificar transação"
+
+#: src/screens/DecodeTx.js:91
+msgid "push"
+msgstr "forçar"
+
+#: src/screens/DecodeTx.js:91
+msgid ""
+"Write your transaction in hex value and click the button to get a human "
+"value description"
+msgstr ""
+"Escreva sua transação em hexadecimal e clique no botão para obter os detalhes"
+
+#: src/screens/DecodeTx.js:93
+msgid "Could not decode this data to a transaction"
+msgstr "Não foi possível decodificar os dados dessa transação"
+
+#: src/screens/LoadWallet.js:115
+msgid "Write the 24 words of your wallet (separated by space)."
+msgstr "Escreva as 24 palavras da sua wallet (separadas por espaço)."
+
+#: src/screens/LoadWallet.js:116
+msgid "Words separated by single space"
+msgstr "Palavras separadas por um espaço"
+
+#: src/components/BackButton.js:33 src/components/PinPasswordWrapper.js:60
+#: src/screens/LoadWallet.js:119 src/screens/NewWallet.js:164
+#: src/screens/NewWallet.js:177
+msgid "Back"
+msgstr "Voltar"
+
+#: src/screens/LoadWallet.js:120
+msgid "Import data"
+msgstr "Importar wallet"
+
+#: src/screens/LoadingAddresses.js:66
+msgid "Loading transactions..."
+msgstr "Carregando transações..."
+
+#: src/screens/LoadingAddresses.js:70
+msgid "Please wait while we load the transactions of all your addresses."
+msgstr "Por favor, aguarde enquanto nós carregamos as suas transações."
+
+#: src/screens/LoadingAddresses.js:71
+msgid ""
+"You will be automatically redirected to the wallet when we finish loading "
+"them."
+msgstr "Você será automaticamente redirecionado para a wallet quando terminar."
+
+#: src/screens/LoadingAddresses.js:72
+#, javascript-format
+msgid "<strong>Addresses found:</strong> ${ this.props.addressesFound }"
+msgstr "<strong>Endereços encontrados:</strong> ${ this.props.addressesFound }"
+
+#: src/screens/LoadingAddresses.js:73
+msgid "<strong>Transactions found:</strong> ${ this.props.transactionsFound }"
+msgstr ""
+"<strong>Transações encontradas:</strong> ${ this.props.transactionsFound }"
+
+#: src/screens/LockedWallet.js:90
+msgid "Your wallet is locked. Please write down your PIN to unlock it."
+msgstr ""
+"Sua wallet está bloqueada. Por favor, entre com o seu PIN para desbloqueá-la."
```

Comment:
I think I prefer "digite" instead of "entre com"

Reply:
Fixed. Thanks!